crates/nu-std/README.md

```diff
@@ -3,36 +3,78 @@
+│ 29 │ std xaccess                 │ Get all xml entries matching simple xpath-inspired query            │
+│ 30 │ std xinsert                 │ Insert new entry to elements matching simple xpath-inspired query   │
+│ 31 │ std xtype                   │ Get type of an xml entry                                            │
+│ 32 │ std xupdate                 │ Update xml data entries matching simple xpath-inspired query        │
+├────┼─────────────────────────────┼─────────────────────────────────────────────────────────────────────┤
+│  # │            name             │                                usage                                │
+╰────┴─────────────────────────────┴─────────────────────────────────────────────────────────────────────╯
 
-## :toolbox: use the standard library in the REPL or in scripts
-in order to "import" the standard library to either the interactive [*REPL*][REPL] of
-`nushell` or inside some `.nu` script, you might want to use the
-[`use`](https://nushell.sh/commands/docs/use.html) command!
-```bash
-use std
 ```
 
-## :pencil2: contribute to the standard library
-- all the commands of the standard_library are located in [`std.nu`](std.nu)
-- the tests are located in files that have a name starting with "test_", e.g. [`test_std.nu`](test_std.nu)
-- a test runner, at [`tests.nu`](tests.nu), allows to run all the tests automatically
+## :toolbox: Using the standard library in the REPL or in scripts
+All commands in the standard library must be "imported" into the running environment 
+(the interactive read-execute-print-loop (REPL) or a `.nu` script) using the
+[`use`](https://nushell.sh/commands/docs/use.html) command.
 
-### :wrench: add new commands
-- add new standard commands by appending to [`std.nu`](std.nu)
-- add associated tests to [`test_std.nu`](tests_std.nu) or preferably to `test_<submodule>.nu`.
-    - define a new exported (!) `test_<feature>` command
-    - import the `assert` functions you need at the top of the functions, e.g. `use std.nu "assert eq"`
+You can choose to import the whole module, but then must refer to individual commands with a `std` prefix, e.g:
+```
+use std
+ . . .
+std log debug "Running now"
+std assert (1 == 2)
+```
 
-### :test_tube: run the tests
-the following call should return no errors
-```bash
-NU_LOG_LEVEL=DEBUG cargo run -- crates/nu-std/tests.nu
+Or you can enumerate the specific commands you want to import and invoke them without the `std` prefix.
 ```
+use std log assert
+. . .
+log debug "Running again"
+assert (2 == 1)
+```
+This is probably the form of import you'll want to add to your `env.nu` for interactive use.
 
-> **Warning**  
-> the `cargo run --` part of this command is important to ensure the version of `nushell` and the version of the library are the same.
+## :pencil2: contribute to the standard library
+You're invited to contribute to the standard library! 
+See [CONTRIBUTING.md](./CONTRIBUTING.md) for details
```

Comment:
maybe a period here for consistency with the rest of the document? :yum:

Reply:
yes.

Reply:
:+1: